lib/SPIRVProducerPass.cpp

```diff
@@ -2084,8 +2091,10 @@ SPIRVID SPIRVProducerPassImpl::getSPIRVType(Type *Ty, bool needs_layout) {
         // Ops[2] = Stride Number(Literal Number)
         Ops.clear();
 
+        // if stride is 1 automatically set it to 2 to avoid stride issues.
```

Comment:
The comment is not correct anymore, please update it.

Reply:
thank you!